src/modules/love/boot.lua

```diff
@@ -45,6 +45,17 @@ function love.boot()
 	-- This is absolutely needed.
 	require("love.filesystem")
 
+	-- nogame.lua returns noGameRestartInfo when
+	-- dropping a file or directory on to its window.
+	if love.restart and type(love.restart) == "table" then
+		local was_no_game = love.restart.noGameRestartInfo
```

Comment:
Maybe this could be `_noGameRestartInfo` (with the underscore prefix) to signify that it's for internal use only